test/powershell/engine/Basic/DefaultCommands.Tests.ps1

```diff
@@ -124,7 +124,7 @@ Describe "Verify approved aliases list" -Tags "CI" {
 "Alias",        "npssc",                            "New-PSSessionConfigurationFile",   $($FullCLR                               ),     "ReadOnly",             "",                     ""
 "Alias",        "nsn",                              "New-PSSession",                    $($FullCLR -or $CoreWindows -or $CoreUnix),     "",                     "",                     ""
 "Alias",        "nv",                               "New-Variable",                     $($FullCLR -or $CoreWindows -or $CoreUnix),     "ReadOnly",             "",                     ""
-"Alias",        "nwsn",                             "New-PSWorkflowSession",            $($FullCLR                               ),     "ReadOnly",             "",                     ""
+"Alias",        "nwsn",                             "New-PSWorkflowSession",            $($FullCLR -or $CoreWindows              ),     "",                     "",                     ""
```

Comment:
Why do we need the change?

Reply:
this alias comes from `PSWorkflow` module which is `incompartible` with PS Core.
This test just loads all stuff from a hardcoded set of modules and checks against this huge table; `PSWorkflow` is one of them. With this PR module can actually be successfully loaded on PS Core and the alias shows up, and the test did not expect this.

Reply:
DefaultCommands.Tests is designed to tracking that is explicitly in PowerShell Core on different platforms and show difference from Windows PowerShell.

If the test failed I guess it is side effect from other tests and just the tests must do right cleanup.